presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaProducerFactory.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+
+import java.util.Properties;
+
+public interface KafkaProducerFactory
+{
+    default KafkaProducer<Object, Object> create(String keySerializer, String messageSerializer)
```

Comment:
Can make return `KafkaProducer<byte[], byte[]>` and handle row encoding in separate layer like it is with `KafkaConsumerFactory`?